vm/compiler/src/ledger/block/header/serialize.rs

```diff
@@ -41,6 +42,7 @@ impl<'de, N: Network> Deserialize<'de> for Header<N> {
                 Ok(Self::from(
                     serde_json::from_value(header["previous_state_root"].clone()).map_err(de::Error::custom)?,
                     serde_json::from_value(header["transactions_root"].clone()).map_err(de::Error::custom)?,
+                    serde_json::from_value(header["coinbase_accumulator_point"].clone()).map_err(de::Error::custom)?,
```

Comment:
same as https://github.com/AleoHQ/snarkVM/pull/1100/files#r996780312, applies to all the clones